frontend/models/product-list/server.ts

```diff
@@ -85,6 +88,11 @@ export async function findProductList(
       productListType === ProductListType.AllParts ||
       productListType === ProductListType.DeviceParts;
 
+   const reusableSections = await findProductListReusableSections({
+      strapiProductList: productList,
+      ancestorHandles: ancestors.map((a) => a.handle),
+   });
```

Comment:
This might be a micro-optimization, but we could `Promise.all()` this with `getProductListChildren()`